hw2/js/dateCalc/datecalc.js

```diff
@@ -0,0 +1,19 @@
+import { DateTime } from "./luxon.js";
```

Comment:
Давай использовать luxon из npm 
```suggestion
import { DateTime } from "luxon";
```
И файлик luxon.js здесь удали

Reply:
Поправил